cmd/pitr/collector/collector.go

```diff
@@ -313,5 +322,9 @@ func readBinlog(file *os.File, pipe *io.PipeWriter, errBuf *bytes.Buffer, binlog
 		pipe.CloseWithError(errors.New("mysqlbinlog error:" + errBuf.String()))
 		return
 	}
-	pipe.Close()
+	err := pipe.Close()
```

Comment:
it always return nil =)

Reply:
nice) Thanks. Removed in https://github.com/percona/percona-xtradb-cluster-operator/pull/768/commits/548df709315de0137e70807e4884404a80524752